### ISSUE TITLE
Rename Fbits to Wake

### ISSUE THREAD

**Issue by Angeesantos** [HTTPArchive/wappalyzer#15]
Hi, Good Afternoon!

I'm Angela and I want to know how you guys can change the name of our technology platform. We've recently undergone a name change, and we would appreciate it if you could kindly update the information to reflect our current name.

Today, at Core Web Vitals Technology Report, we appear as Fbits, being necessary to change for **Wake**.

![image](https://github.com/HTTPArchive/wappalyzer/assets/160787180/945f91f8-a6dd-4772-a082-3cd3fde562c4)

We understand the importance of maintaining accurate and up-to-date information on your service and believe that this change will help ensure the accuracy of our representation. We kindly request your assistance in making this update.

Furthermore, all websites listed under the FBITS technology are owned by us.

Please inform us about the necessary steps and the information you require from us to facilitate the update of our platform's name. We are committed to providing any documentation or details that may be needed promptly.

We appreciate your attention to this matter and look forward to your response. Thank you for your ongoing support.

**Comment by rviscomi** [comment]
The Fbits detection is in this file:

https://github.com/HTTPArchive/wappalyzer/blob/main/src/technologies/f.json#L449-L458

Is the `fbits` JavaScript variable still the best way to detect Wake?

**Comment by Angeesantos** [comment]
In this case we need to change the informations as:

 "*Wake*": {
    "cats": [
      6
    ],
    "icon": "*Wake.png*", *(file attached)*
    "js": {
      "fbits": ""
    },
    "website": "*https://wake.tech <https://wake.tech>*"
  },

Sorry for my unknowledge, but what the "cats:6" means?

**Comment by Angeesantos** [comment]
Rick, can we insert more information like description?

**Comment by rviscomi** [comment]
> Sorry for my unknowledge, but what the "cats:6" means?

Category 6 is Ecommerce, so given that this is a rename I assume that doesn't change.

> Rick, can we insert more information like description?

Yes, all of the supported properties are documented in the [README](https://github.com/HTTPArchive/wappalyzer?tab=readme-ov-file#json-fields). Most things have no effect anywhere on the HTTP Archive side (including the Tech Report dashboard) but we do plan to start incorporating the description field, so I'd encourage you to suggest one.

**Comment by Angeesantos** [comment]
Hi Rick!

I received an e-mail from Barry who told that, for a quicker action, I should move infos from f.json to w.json file and I just did it.

 ```
 "Wake": {
    "cats": [
      6
    ],
    "description": "Wake is an ecommerce software with integrated digital solutions to manage online stores.",
    "headers": {
      "X-Powered-By": "Wake"
    },
    "icon": "Wake.png",
    "js": {
      "wake": ""
    },
    "saas": true,
    "website": "https://wake.tech/"
```
Can you check it please?
https://github.dev/HTTPArchive/wappalyzer/blob/5e8d4898f4c0260343a19db532f5a467fcb39dfc/src/technologies/w.json

**Comment by rviscomi** [comment]
Yeah you'll need to fork the repository and submit a pull request to make any direct changes.

I do have some reservations about the `wake` JS variable detection. That might be so generic that it leads to false positives. Probably not a big deal, but if the `X-Powered-By` header is reliable enough, maybe we use that alone?

For testing, do you also have a few example URLs of pages that use Wake?

**Comment by Angeesantos** [comment]
Did it!

About the header, you mean use it without "wake" mention?

Here are some URLs os pages that use Wake:

- https://www.stanley1913.com.br/ 
- https://www.puravida.com.br/
- https://www.b2cstore.com.br/
- https://www.dlkmodas.com.br/
- https://www.newlentes.com.br/
- https://www.abcdaconstrucao.com.br/
- https://www.esportelegal.com.br/
- https://www.suggar.com.br/
- https://www.camys.com.br/
- https://www.justapprove.com.br/
- https://www.mundotupperware.com.br/
- https://www.forlabexpress.com.br/
- https://www.marisa.com.br/
- https://www.yeesco.com.br/

**Comment by rviscomi** [comment]
From a quick test of loading the sample pages, it looks like the `X-Powered-By` header is only present 9/14 times and the `wake` JavaScript variable is _never_ set. Does that sound right?

Site | X-Powered-By: Wake | `wake`
-- | -- | --
https://www.stanley1913.com.br/ | ✅ |  
https://www.puravida.com.br/ |   |  
https://www.b2cstore.com.br/ | ✅ |  
https://www.dlkmodas.com.br/ | ✅ |  
https://www.newlentes.com.br/ | ✅ |  
https://www.abcdaconstrucao.com.br/ | ✅ |  
https://www.esportelegal.com.br/ | ✅ |  
https://www.suggar.com.br/ | ✅ |  
https://www.camys.com.br/ | ✅ |  
https://www.justapprove.com.br/ | ✅ |  
https://www.mundotupperware.com.br/ |   |  
https://www.forlabexpress.com.br/ |   |  
https://www.marisa.com.br/ |   |  
https://www.yeesco.com.br/ |   |

**Comment by Angeesantos** [comment]
Rick, following your review, can you check if the `Fbits` JavaScript variable is set in these URLs please?

**Comment by rviscomi** [comment]
It's not consistently used, and it doesn't add any coverage missed by the `X-Powered-By` header

Site | X-Powered-By: Wake | `wake` | `Fbits`
-- | -- | -- | --
https://www.stanley1913.com.br/ | ✅ |   |  
https://www.puravida.com.br/ |   |   |  
https://www.b2cstore.com.br/ | ✅ |   |  
https://www.dlkmodas.com.br/ | ✅ |   |  
https://www.newlentes.com.br/ | ✅ |   | ✅
https://www.abcdaconstrucao.com.br/ | ✅ |   |  
https://www.esportelegal.com.br/ | ✅ |   | ✅
https://www.suggar.com.br/ | ✅ |   | ✅
https://www.camys.com.br/ | ✅ |   |  
https://www.justapprove.com.br/ | ✅ |   | ✅
https://www.mundotupperware.com.br/ |   |   |  
https://www.forlabexpress.com.br/ |   |   |  
https://www.marisa.com.br/ |   |   |  
https://www.yeesco.com.br/ |   |   |

**Comment by Angeesantos** [comment]
Ok Rick! I forked the repository again and submited a pull request to make changes by including Wake infos and deleting Fbits from the file.

```
"Wake": {
   "cats": [
     6
   ],
   "description": "Wake is an ecommerce software with integrated digital solutions to manage online stores.",
   "headers": {
     "X-Powered-By": "Wake"
   },
   "icon": "Wake.png"
   },
   "saas": true,
   "website": "https://wake.tech/"
```

**Comment by Angeesantos** [comment]
Hi Rick, how are you?

Can you please check the request I did? So we can start visualizing Wake infos instead of Fbits, reinforcing our position in the e-commerce ranking.

**Comment by max-ostapenko** [comment]
fixed in #19
